packages/codebyte-editor/.storybook/main.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
```

Comment:
Let's stick to just one story extension. No need to allow multiple.

In general, let's try to use the same everything as Gamut.